avocado/core/nrunner.py

```diff
@@ -448,6 +448,25 @@ def run(self):
 RUNNERS_REGISTRY_PYTHON_CLASS['exec-test'] = ExecTestRunner
 
 
+class RequirementRunner(ExecRunner):
+    """
+    Runner for standalone executables treated as requirements
```

Comment:
I'm trying to understand why we need a special runner just for this. Maybe you could explain better on this docstring?

Reply:
There is a couple of reason that I don't think makes sense to add to the docstring.

The `ExecRunner` do not have the `most_current_execution_state` implementation. I thought about using it for future extension of the logging capabilities.

I started with an implementation that used the `ExecTestRunner` but reached some problems because of the `skip` status, which is not desired on a requirements resolution.

To avoid the problems described above, I thought it would be worth creating a new runner and extend it in the future based on the support for new requirements types.